File: app/src/main/java/com/akameko/testforlifehackstudio/repository/CompanyListItem.java
package com.akameko.testforlifehackstudio.repository;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class CompanyListItem implements Serializable {
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("img")
    @Expose
    private String img;

    public CompanyListItem(Integer id, String name, String img) {
        this.id = id;
        this.name = name;
        this.img = img;
    }

    public Integer getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getImg() {
        return img;
    }

    @Override
    public String toString() {
        return "CompanyListItem{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", img='" + img + '\'' +
                '}';
    }
}

File: app/src/main/java/com/akameko/testforlifehackstudio/repository/CompanyListItem.kt
package com.akameko.testforlifehackstudio.repository

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class CompanyListItem(@field:Expose @field:SerializedName("id") val id: Int, @field:Expose @field:SerializedName("name") val name: String, @field:Expose @field:SerializedName("img") val img: String) : Serializable {

    override fun toString(): String {
        return "CompanyListItem{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", img='" + img + '\'' +
                '}'
    }

}